pkg/client/powervs_client.go

```diff
@@ -419,3 +406,13 @@ func getCustomEndPointKeys(customEndpointsMap map[string]string) []string {
 	}
 	return keys
 }
+
+// GetRegion return the PowerVS region
+func GetRegion() string {
+	return getEnvironmentalVariableValue(regionEnvironmentalVariable)
+}
+
+// GetZone return the PowerVS zone
+func GetZone() string {
+	return getEnvironmentalVariableValue(zoneEnvironmentalVariable)
+}
```

Comment:
Instead of set/get from env, can we get it from `powerVSClient` struct?

Reply:
Updated the methods to retrieve information from `powerVSClient`